deps.bat

```diff
@@ -269,9 +269,9 @@ For more info, see:
 
 (defn- home-dir []
   (if windows?
-    ;; workaround for https://github.com/oracle/graal/issues/1630
-    (*getenv-fn* "userprofile")
-    (System/getProperty "user.home")))
+    (or (*getenv-fn* "home")
```

Comment:
This should probably be capitalized: `HOME`.
Similar to `USERPROFILE`. Environment variables are case sensitive.

Reply:
Ah, you got a point! I just blindly copied the lowercase spelling from the `userprofile` in the windows branch. Apparently there, environment variables *are* case insensitive? :thinking: Anyhow, I also uppercased them there for consistency with the original script. Let's see how it goes :crossed_fingers:

Reply:
What windows branch?

Reply:
as in the truthy branch of the `(if windows? ...)` form.

Hold on, I forgot to re-generate `deps.clj`... coming up.

Reply:
I don't know what you are referring to, maybe you can paste a Github permalink with a line number.

Reply:
It's the lines this patch is touching: https://github.com/borkdude/deps.clj/pull/115/files#diff-de1d8b28c217e59dca32b58323dd56ab342bbcb074663d89807109756be86dfdR263-R265 - as you can see, `userprofile` used to be lowercased before.